src/components/home/main/MainContent.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-extra-boolean-cast */
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Smilies from 'assets/smilies.svg';
+import { ReactComponent as ProfilePlaceholder } from 'assets/profile_placeholder.svg';
+import FootballTopicIcon from 'assets/football_topic_icon.png';
+import MusicTopicIcon from 'assets/music_topic_icon.png';
+import TravelTopicIcon from 'assets/travel_topic_icon.png';
+import { useSession } from 'hooks';
+
+import './mainContent.scss';
+
+const MainContent = () => {
+  const { user } = useSession();
+
+  return (
+    <div className="main-container">
+      <h2 className="main-title">
+        <FormattedMessage id="home.bold_target" />
+      </h2>
+      <div className="home-body">
+        <div className="profile-container">
+          <div className="profile-picture-background">
+            <ProfilePlaceholder />
+          </div>
+          {!!user.username ? (
```

Comment:
```suggestion
          {user.username ? (
```
to remove /* eslint-disable no-extra-boolean-cast */  line 

Because you are checking the bool value already with the conditional, sorry that I didn't notice that